projects/1.16/assets/astikorcarts/astikorcarts/lang/zh_cn.json

```
@@ -1 +1,17 @@
-{}
\ No newline at end of file
+{
+  "item.astikorcarts.supply_cart": "马车",
+  "item.astikorcarts.plow": "犁",
+  "item.astikorcarts.animal_cart": "牲畜马车",
+  "item.astikorcarts.wheel": "轮子",
+  "itemGroup.astikorcarts": "马车",
```

```suggestion
  "itemGroup.astikorcarts": "AstikorCarts",
```
astikor可能是自造词，建议不翻mod名